modules/navigation/nav_map.cpp

```diff
@@ -606,6 +606,11 @@ void NavMap::sync() {
 		for (size_t poly_id(0); poly_id < polygons.size(); poly_id++) {
 			gd::Polygon &poly(polygons[poly_id]);
 
+			// skip triangles which are not triangles after aliging vertices towards cells (see issue #56786)
```

Comment:
Our preferred comment style is to use complete sentence formatting (also typo fix, and change to make issue references easier to grep):
```suggestion
			// Skip triangles which are not triangles after aligning vertices towards cells (see GH-56786).
```

Reply:
done, thanks for suggestion!